roxie/ccd/ccdserver.cpp

```diff
@@ -2755,6 +2755,15 @@ class CRoxieServerMultiInputBaseActivity : public CRoxieServerActivity
             return NULL;
     }
 
+    virtual void stop() override
```

Comment:
@richardkchapman 
Some of the base classes already have some code like this e.g.
NonEmpty, Case - conditionally depending on what has been called already

As long as it is ok to call stop() multiple times then it shouldn't make things worse, but not convinced it is the "coorect" fix.  Worth looking a bit more?

Reply:
I did spend a little while looking at whether it was necessary to check that we don't call stop more than once. In general I don't think it's possible to be 100% sure that you do not do so, nor is it necessary given there is code in several places explicitly to cope with having called stop more than once.